__tests__/caculateRevenue.test.js

```diff
@@ -0,0 +1,34 @@
+import { calculateRevenue } from '../src/domain/calculateRevenue.js';
+
+describe('로또 수익률 계산 테스트', () => {
+  let purchasePrice;
+
+  beforeEach(() => {
+    purchasePrice = 1000;
+  });
+  test('로또 당첨 개수가 6개인 경우 수익률 ', () => {
+    const matchCounts = [0, 0, 0, 0, 0, 0, 1, 0];
+
+    const expectedResult = Number(((2000000000 / purchasePrice) * 100).toFixed(1));
```

Comment:
큰 숫자에는 numeric separator를 사용해보시면 좋을거 같아요
```suggestion
    const expectedResult = Number(((2_000_000_000 / purchasePrice) * 100).toFixed(1));
```

Reply:
오.. numeric separator 이라는게 있군요. 좋은 정보 감사합니다!